### DISCUSSION TITLE
Baseline Contribution Error

### DISCUSSION THREAD

**Discussion by smmalik98** [google/lightweight_mmm#105] [General]
I have been trying to plot the baseline media contribution but it has been failing since one of my media contribution has positive and negative values. This problem started when I reduced the number of columns. Initially my data had 7 media columns and 35 extra features with around 157 rows (weekly data). But now it has reduced to 5 media columns and 15 extra features. I have also added the screenshot of the error that I am getting. 
I need help in interpreting what is wrong with the model?
<img width="1690" alt="Screen Shot 2022-11-20 at 11 48 02 AM" src="https://user-images.githubusercontent.com/97324716/202917839-ae9ea69c-f3b3-41ac-adef-7acb85f6040e.png">

**Comment by pabloduque0** [comment]
Sorry we missed this one initially.

This was answered in #107
